src/core/Form/FilterInput/FilterInput.baseStyles.tsx

```diff
@@ -0,0 +1,77 @@
+import { css } from 'styled-components';
+import { withSuomifiTheme, TokensAndTheme } from '../../theme';
+import { input, containerIEFocus, font } from '../../theme/reset';
+import { absoluteFocus } from '../../theme/utils';
+
+export const baseStyles = withSuomifiTheme(
+  ({ theme }: TokensAndTheme) => css`
+    & .fi-filter-input {
+      ${font({ theme })('bodyText')}
+      display: inline-block;
```

Comment:
what happens if fi-filter-input is the only place with display set and someone gives the outermost div a style with display: flex? Should the display value be set deeper in the structure, let's say in the wrapper?

Reply:
This is now done in wrapper.